detection_rules/rule.py

```diff
@@ -230,6 +230,16 @@ class AlertSuppressionDuration:
 
     group_by: List[definitions.NonEmptyStr]
     duration: Optional[AlertSuppressionDuration] = field(metadata=dict(metadata=dict(min_compat="8.7")))
+    missing_fields_strategy: Optional[definitions.AlertSuppressionMissing] = field(
```

Comment:
I don't think this is actually `Optional`, right? It requires one or the other

Reply:
So, there are 2 ways we can handle the fact that this only exists in 8.7+, whereas full alert suppression exists for 8.6+

A: drop support for 8.6 - technically this feature is still in technical preview and so this is a valid option

B: 
* leave it as `Optional`
* add a custom marshmallow validation method the: `<8.7 not allowed, >=8.7 required`